packages/cli/src/commands/inspect/index.ts

```diff
@@ -174,26 +233,11 @@ export default async function inspect(client: Client) {
     print(indent(routesList(routes), 4));
     print(`\n\n`);
   }
-
-  return 0;
 }
 
-function stateString(s: Deployment['readyState']) {
-  const CIRCLE = '● ';
-  const sTitle = s && title(s);
-  switch (s) {
-    case 'INITIALIZING':
-    case 'BUILDING':
-      return chalk.yellow(CIRCLE) + sTitle;
-    case 'ERROR':
-      return chalk.red(CIRCLE) + sTitle;
-    case 'READY':
-      return chalk.green(CIRCLE) + sTitle;
-    case 'QUEUED':
-      return chalk.gray(CIRCLE) + sTitle;
-    case 'CANCELED':
-      return chalk.gray(CIRCLE) + sTitle;
-    default:
-      return chalk.gray('UNKNOWN');
+function exitCode(state: Deployment['readyState']) {
+  if (state === 'ERROR' || state === 'CANCELED') {
+    return 1;
   }
+  return 0;
 }
```

Comment:
Shouldn't this be returning `0` all the time if you don't want to change how status code is returned?